piet-web/examples/basic/src/lib.rs

```diff
@@ -44,8 +57,10 @@ pub fn run() {
         .dyn_into::<web_sys::CanvasRenderingContext2d>()
         .unwrap();
 
-    canvas.set_width(canvas.offset_width() as u32);
-    canvas.set_height(canvas.offset_height() as u32);
+    let dpr = window().map(|w| w.device_pixel_ratio()).unwrap_or(1.0);
```

Comment:
nit: we already grab the window above and crash if it doesn't exist. We could just use that window and avoid having to `map`